Competition2017/.classpath

```diff
@@ -12,5 +12,9 @@
 	<classpathentry kind="lib" path="/SeriouslyCommonLib/lib/WPILib-sources.jar"/>
 	<classpathentry kind="lib" path="/SeriouslyCommonLib/lib/WPILib.jar" sourcepath="/SeriouslyCommonLib/lib/WPILib-sources.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="var" path="wpilib" sourcepath="wpilib.sources"/>
```

Comment:
Are you sure? I've been working with this project and AFAIK it hasn't don't this... same with some other Eclipse projects that have a similar JAR configuration. Is there a specific action that causes it?